Gems/OpenXRVk/Code/Source/OpenXRVkInstance.cpp

```diff
@@ -412,6 +415,16 @@ namespace OpenXRVk
         return m_supportedXRDevices[m_physicalDeviceActiveIndex];
     }
 
+    void Instance::FilterAvailableExtensions(GladVulkanContext& context) const
+    {
+        // In some cases (like when running with the GPU profiler on Quest2) the extension is reported as available
```

Comment:
Can you add that this was specifically when we try to use "adb shell ovrgpuprofiler -e" so it is clear when this code applies